cmd/operator/deploy/crds/monitoring.googleapis.com_operatorconfigs.yaml

```diff
@@ -304,11 +332,6 @@ spec:
                       - port
                       type: object
                     type: array
-                  enableManagedAlertManager:
```

Comment:
I'm assuming this change is not yet released? Otherwise I think it's a breaking change.

Reply:
correct this is not yet released, so it won't be breaking. we got it in time :)